### PR TITLE
Update the link for contributor swag

### PR DESCRIPTION
**Pull request by tfny**
S/A

### What does this PR do?

corrects the link for folks to let us know when they create content.  This link was over a year out of date.

### Motivation

I like to be helpful.  It makes me feel warm and fuzzy, but at the same time powerful.  Like a god, but of something very tiny and not very important.  Goddess of the specks of dust that no one sees, but also will never coalesce into something more powerful.  Or maybe, goddess of the trailing space.  

Mostly, though, I was motivated by my job. 

### More

- [ ] Added/updated tests
- [ ] Added/updated documentation

### CODE REVIEW

File: docs/content/contributing/advocating.md [hunk 1]
```diff
@@ -10,6 +10,6 @@ Spread the Love & Tell Us about It
 
 There are many ways to contribute to the project, and there is one that always spark joy: when we see/read about users talking about how Traefik helps them solve their problems.
 
-If you're talking about Traefik, [let us know](https://blog.traefik.io/spread-the-love-ba5a40aa72e7) and we'll promote your enthusiasm!
+If you're talking about Traefik, [let us know]([https://blog.traefik.io/spread-the-love-ba5a40aa72e7](https://traefik.io/submit-my-contribution/)) and we'll promote your enthusiasm!
```
Comment (ldez) [inline docs/content/contributing/advocating.md:13]:
```suggestion
If you're talking about Traefik, [let us know](https://traefik.io/submit-my-contribution/) and we'll promote your enthusiasm!
```

**Review by rtribotte — APPROVED** [review]
LGTM 👌

**Review by tomMoulard — APPROVED** [review]
LGTM :ok_hand:

(we might still want to update the blog post)

**Review by kevinpollet — APPROVED** [review]
Thanks 🤩

**Pull request merged**